services/esSearcherPreprocessorSvc.js

```diff
@@ -61,6 +62,26 @@ angular.module('o19s.splainer-search')
         }
 
         searcher.queryDsl   = queryDsl;
+      };
+
+      var prepareGetRequest = function (searcher) {
+        searcher.url = searcher.url + '?q=' + searcher.queryText;
+      };
```

Comment:
Just FYI, you can probably also do this with a POST request (using the query_string or simple_query_string ES query).  I don't know if that helps simplify anything.